prisma/schema.sql

```diff
@@ -89,7 +89,7 @@ CREATE TABLE classes (
 CREATE TABLE parents (
   id SERIAL PRIMARY KEY NOT NULL,
   phone_number VARCHAR(50) NOT NULL,
-  is_low_income BOOLEAN DEFAULT false,
+  is_low_income BOOLEAN,
```

Comment:
@MatoPlus Can you explain why false isn't equivalent to the "pending" state?

Reply:
My understanding is that we have three states, denied, pending, and approved. This reflects our design for internal but not external (atm since we only have pending and approved for external).